index.js

```diff
@@ -108,9 +108,13 @@ export const Link = (props) => {
       )
         return;
 
+      onClick && onClick(event);
+      if (event.defaultPrevented) {
+        return;
+      }
+
       event.preventDefault();
       navRef.current();
-      onClick && onClick(event);
```

Comment:
I'm trying to think about any possible use case when the order of `onClick` and `navRef()` would matter. It doesn't seem so at the first glance, but maybe @Tomas2D you have some thoughts?